### ISSUE TITLE
vets-website prep debugging

### ISSUE THREAD

**Issue by timwright12** [department-of-veterans-affairs/va.gov-team#21520]
This is a ticket to run planning simulations inside vets-website to understand what needs to happen when we remove the content-build from the process.

**Currently we know:**

- Nightwatch tests fail
- Nightwatch a11y tests fail (will be removed)
- Cypress tests fail
- Some assets will need to be moved to content-build, others will need to stay in vets-website

The build can be updated to use a --scaffold option to pass some tests, but this script needs to be updated to allow the rest oft he tests to pass or be migrated into the content-build repo.

**Comment by timwright12** [comment]
Some nightwatch e2e tests pass, but 74 are failing, so we need to figure out which ones need to be moved over to the content build repo and which ones need to be fixed.

**Comment by theodur** [comment]
The two failing Cypress tests that were unaccounted for in the application build have been fixed in [department-of-veterans-affairs/vets-website#16477](https://github.com/department-of-veterans-affairs/vets-website/pull/16477):
- `src/applications/find-forms/tests/e2e/find-forms-required.cypress.spec.js`
- `src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/tests/index.cypress.spec.js`

**Comment by theodur** [comment]
We still have 15 nightwatch tests [failing](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/fix-cypress-tests-for-apps/1/tests) in the `nightwatch-e2e` stage in Jenkins when using the `--scaffold` option and omitting the content build. I've started a [draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16490) for fixing these failing tests.

**Comment by timwright12** [comment]
1. Only a handful of tests still failing, will ID tests that need to be moved to content-build
2. Need to address `src/site/assets` w/o breaking teamsites
3. Need to get content-build generating a release for prod

**Comment by theodur** [comment]
The work for fixing the nightwatch test has been moved to this [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16494) instead.

Currently we only have two failing nightwatch tests:
- `src/site/tests/home/00-required.e2e.spec.js`
- `src/platform/site-wide/tests/accessible-modal.e2e.spec.js`

Both of these tests will be moved to the `content-build` repo and removed from the `vets-website` repo, so they are not a problem.

We have 4 failing Cypress tests:
- `src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js`
- `src/applications/search/tests/e2e/00-required.cypress.spec.js`
- `src/site/layouts/tests/vamc/location_home.cypress.spec.js`
- `src/site/layouts/tests/vamc/system_home.cypress.spec.js`

Based on [this conversation](https://dsva.slack.com/archives/C01512KN35G/p1616682179059200), the last two Cypress tests listed above will be replaced with unit tests, so they are not a concern. The first two Cypress tests will need to be resolved, and are currently being worked on.

**Comment by theodur** [comment]
We now have all the Cypress tests needed passing in the draft PR. So the only two failing Cypress tests are the ones that will be converted to unit tests by the Facilities team.

**Comment by theodur** [comment]
The [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16494) for fixing the nightwatch and Cypress tests has been merged. The `scaffoldRegistry` and `facilitySidebar` paths will need to updated before the content build release. This should be added to the launch checklist ticket when created.

**Comment by timwright12** [comment]
Manually migrating new liquid unit tests today.

Also need to confirm that preview functionality still works

**Comment by timwright12** [comment]
Discovered that preview functionality in vets-website isn't working after content separation, spoke with public websites and, while not a har requirement, this is a desired feature to retain, so  we will work to restore it.

**Comment by timwright12** [comment]
Working on syncing repo permissions.

**Comment by meganhkelley** [comment]
Feedback from VFS teams based on testing in dev: 

* [Health Care Experience team](https://dsva.slack.com/archives/C5HP4GN3F/p1618327777076600?thread_ts=1617738055.063600&cid=C5HP4GN3F)
* [Facilities team](
* [VAOS](https://dsva.slack.com/archives/C5HP4GN3F/p1618333672077700?thread_ts=1617738055.063600&cid=C5HP4GN3F)
* [eBenefits](https://dsva.slack.com/archives/C01S39QMSSV/p1619184300055300)

**Comment by jhonnyoddball** [comment]
## Health Care Experience team

[Issue](https://dsva.slack.com/archives/C5HP4GN3F/p1618327777076600?thread_ts=1617738055.063600&cid=C5HP4GN3F)

### Outcome

I contacted Mark Dewey and confirmed that it is no longer an issue

**Comment by jhonnyoddball** [comment]
## VAOS

[Issue](https://dsva.slack.com/archives/C5HP4GN3F/p1618333672077700?thread_ts=1617738055.063600&cid=C5HP4GN3F)

### Outcome

I contacted Jeff Balboni and confirmed that it’s related to `vets-api` environment config. The front end of VAOS loads and appears to work correctly

**Comment by jhonnyoddball** [comment]
Talked to Michael PS and Jason Wolf and there are no related known issues in dev nor staging due to `content-build`
